client/components/jetpack-header/partner-logo-group.jsx

```diff
@@ -18,7 +18,7 @@ class PartnerLogoGroup extends PureComponent {
 
 	static propTypes = {
 		partnerName: PropTypes.string,
-		width: PropTypes.number,
+		width: PropTypes.oneOfType( [ PropTypes.number, PropTypes.string ] ),
```

Comment:
We won't need to support a string `width` prop if we apply the width/max-width or apply some padding/margin by using CSS.